blockchain/vm/jump_table.go

```diff
@@ -63,11 +63,23 @@ type operation struct {
 var (
 	ConstantinopleInstructionSet = newConstantinopleInstructionSet()
 	IstanbulInstructionSet       = newIstanbulInstructionSet()
+	LondonInstructionSet         = newLondonInstructionSet()
 )
 
 // JumpTable contains the EVM opcodes supported at a given fork.
 type JumpTable [256]*operation
 
+// newLondonInstructionSet returns the frontier, homestead, byzantium,
+// constantinople, istanbul, petersburg, berlin and london instructions.
+func newLondonInstructionSet() JumpTable {
+	instructionSet := newIstanbulInstructionSet()
+	// TODO-klaytn: base fee opcode will be enabled
+	//enable3198(&instructionSet) // Base fee opcode https://eips.ethereum.org/EIPS/eip-3198
+	return instructionSet
+}
+
+// newIstanbulInstructionSet returns the frontier, homestead, byzantium,
+// constantinople, istanbul and petersburg instructions.
```

Comment:
Do we have petersburg instructions?

Reply:
@jeongkyun-oh `Petersburg` has been applied to klaytn code.